demo/vl-form-validation.html

```diff
@@ -219,6 +219,92 @@
                 </div>
             </form>
         </vl-demo>
+
+        <vl-demo data-vl-title="Formulier validatie met foutmelding in error-placeholder-element">
```

Comment:
Nodig om een heel form te maken om dat te demonstreren?

Reply:
ja, ik wilde toch testen dat de foutmelding voor elke type validatie ook getoond wordt als die uit de placeholder komt.

Reply:
goh ja, tis niet fout natuurlijk. misschien heeft @coemans nog een mening

Reply:
Het lijkt me in deze overkill gezien de code wijzigingen die niet afwijken per type. De e2e en unit testen lijken me dus overbodig. Ze zijn al geschreven en zolang ze niet onnodig onderhouden moeten worden, kunnen ze wat mij betreft blijven staan.